src/aiNpc/Stats/Segment.ts

```diff
@@ -0,0 +1,126 @@
+import { getUserData } from "@decentraland/Identity"
+import { AnalyticsLogLabel, SKIP_ANALYTICS } from "./AnalyticsConfig";
+import { getRealm } from "src/modules/realmData";
+
+const sceneId: string = "genesis_plaza"
+const inSeconds: boolean = false
+
+var segment: Segment = null
+
+export function getSegment() {
+  if (segment) return segment;
+
+  segment = new Segment('CH4D0eauyRqTDYnqhI0jw1PVPLmaJENj')
+  return segment
+}
+
+export async function sendTrackOld(eventName: string,) {
+  const doc: any = {
+    sceneId: sceneId,
+    playTime: Date.now(),
+    exactPosition: Camera.instance.worldPosition,
+    position: Math.floor(Camera.instance.worldPosition.x / 16) + "," + Math.floor(Camera.instance.worldPosition.z / 16)
+  }
+  await getSegment().track(eventName, doc)
+}
+
+export async function sendTrack(trackEvent: string,
+  elementType: string,
+  elementId: string,
+  instance: string,
+  event: string,
+  selection: string,
+  durationTime: number) {
+
+    const realm = await getRealm()
+
+    const doc: any = {
+      sceneId: sceneId,
+      realm: realm,
+      spanId: instance,
+      elementType: elementType,
+      elementId: elementId,
+      event: event,
+      selection: selection,
+      durationTime: inSeconds ? durationTime * 0.001 : durationTime,
+
+      playTime: Date.now(),
```

Comment:
I think playTime == from start till now,  not the current moment.  that is tracked by  timestamp: new Date().toJSON(),

https://github.com/decentraland/dcl-onboarding/blob/main/src/genesis/stats/segment.ts#L14